ssh/tailssh/tailssh.go

```diff
@@ -861,6 +861,9 @@ func (c *conn) newSSHSession(s ssh.Session) *sshSession {
 // isStillValid reports whether the conn is still valid.
 func (c *conn) isStillValid() bool {
 	a, localUser, err := c.evaluatePolicy(c.pubKey)
+	if sshVerboseLogging() {
```

Comment:
use `ss.vlogf`

Reply:
This is on the conn, there can be multiple sessions and those structures are all guarded by Mu

Reply:
:+1: